Beginner example documentation.

mehr0096 opened the pull request.

mehr0096 committed:
updated beginner

mehr0096 committed:
added link references to doc

mehr0096 committed:
Merge branch 'develop' into beginner-example

mehr0096 committed:
updated text and references

mehr0096 committed:
added photos for beginner rst

mehr0096 committed:
Merge branch 'develop' into beginner-example

mehr0096 committed:
updated channel button name

mehr0096 committed:
updated channel checkbox and exASLM documentation

mehr0096 committed:
changed continuous scan back to quotes to match mesoSPIM-BT case study

mehr0096 committed:
fixed link errors and typos

zacsimile commented on docs/source/beginner.rst:
```diff
@@ -1,3 +1,151 @@
+==============================
+
+#. Load the sample into the microscope chamber.
+#. open navigate software by opening an anaconda prompt, activating navigate by entering the command ``conda activate navigate`` into the command window, press :guilabel:`Enter` and subsequently typing ``navigate`` and press :guilabel:`Enter` into the anaconda command window.
```
For keyboard commands, please use `:kbd:` instead of `:guilabel:`. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html for more details on how these roles work.

zacsimile commented on docs/source/beginner.rst:
```diff
@@ -1,3 +1,151 @@
+
+.. image:: images/beginner/channel-selector.png
+
+#. Select the appropriate filter using the :guilabel:`filter dropdown menu` for the selected channel.
```
Please ensure GUI labels match the GUI. In this case, the GUI reads `Filter` and "dropdown menu" is descriptive.

zacsimile commented on docs/source/beginner.rst:
```diff
@@ -1,3 +1,151 @@
+#. Load the sample into the microscope chamber.
+#. open navigate software by opening an anaconda prompt, activating navigate by entering the command ``conda activate navigate`` into the command window, press :guilabel:`Enter` and subsequently typing ``navigate`` and press :guilabel:`Enter` into the anaconda command window.
+
+.. image:: images/beginner/open-navigate.png
```
This needs to be indented to the same level as the rest of the bullet point in order to prevent the numbering in the list from re-starting.

AdvancedImagingUTSW commented:
@mehr0096, if you could make these changes and make another push, I'd appreciate it.

mehr0096 committed:
Merge branch 'develop' into beginner-example

mehr0096 committed:
updated keyboard labels and numbering and fixed some typos

mehr0096 committed:
fixed typos and links

mehr0096 committed:
updated anaconda commands

mehr0096 committed:
fixed more typos

zacsimile commented on docs/source/beginner.rst:
```diff
@@ -1,3 +1,153 @@
+==============================
+
+#. Load the sample into the microscope chamber.
+#. open navigate software by opening an anaconda prompt
```
"open" should be "Open". "anaconda prompt" should be "Anaconda Prompt". Etc. throughout the document

AdvancedImagingUTSW committed:
Update beginner.rst

I tried to jump in a bit to provide some formatting, but I find the flow somewhat awkward and repetitive. For example, we tell them to set the defocus, interval, etc., multiple times.

mehr0096 committed:
fixed more typos

AdvancedImagingUTSW committed:
Beginner... More.

more changes. Still not done.

AdvancedImagingUTSW committed:
Update beginner.rst

zacsimile committed:
Tweaks
